tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,68 @@
+const AuthorHome = require('../pages/AuthorHome.page');
+const CreateQuestionnairePage = require('../pages/CreateQuestionnaire.page');
+const DesignQuestionnairePage = require('../pages/DesignQuestionnaire.page');
+
+const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
+
+describe('eQ Services Smoke Test', () => {
+  beforeAll(() => {
```

Comment:
This `beforeAll` is doing too much.
Which is probably hidden by the fact that there is only 1 test.
We possibly don't need a before all if we only have one scenario